config/routes.rb

```diff
@@ -7,6 +7,10 @@
     resources :assignments
   end
 
+  resources :grades, only: [:create, :update]
+  delete '/grades', to: 'grades#destroy'
```

Comment:
Add destroy in above resource instead

Reply:
I had tackled the delete route in a similar fashion in featured circuit management as well -  https://github.com/CircuitVerse/CircuitVerse/pull/261#issuecomment-478684132
😅

Reply:
sure